PhotoBooth/PhotoShootSetupCardView.swift

```diff
@@ -0,0 +1,308 @@
+import UIKit
+
+class PhotoShootSetupCard: UIView {
+
+    let fontRegular = "AvenirNext-Regular"
+    let fontBold = "AvenirNext-Bold"
```

Comment:
maybe move this to be an extension of UIFont, that way you can re-use it later on.
```swift
enum FontType: String {
    case regular (or whatever you want to name it) = "AvenirNext-Regular"
// ect put the rest here
}

extension UIFont {
    static func regularFont(size: CGFloat) -> UIFont {
        return UIFont.appFont(type: .regular, size: size)
    }

    static func boldFont(size: CGFloat) -> UIFont {
        return UIFont.appFont(type: .bold, size: size)
    }

    static func appFont(type: FontType, size: CGFloat) -> UIFont {
        return UIFont(name: type.rawValue, size: size)!
    }
}
```